sys/net/network_layer/ng_ipv6/ng_ipv6.c

```diff
@@ -135,6 +135,43 @@ static void *_event_loop(void *args)
     return NULL;
 }
 
+#ifdef MODULE_NG_SIXLOWPAN
+static void _send_to_iface(kernel_pid_t iface, ng_pktsnip_t *pkt)
+{
+    ng_ipv6_netif_t *if_entry = ng_ipv6_netif_get(iface);
+
+    ((ng_netif_hdr_t *)pkt->data)->if_pid = iface;
+
+    if ((if_entry != NULL) && (if_entry->flags & NG_IPV6_NETIF_FLAGS_SIXLOWPAN)) {
```

Comment:
Can you still explain this? What does NULL as a return value for `ng_ipv6_netif_get(iface);` indicate here?